doc/operator-scope.md

```diff
@@ -100,4 +139,21 @@ With the above changes the specified manifests should look as follows:
       Spec   MemcachedSpec   `json:"spec,omitempty"`
       Status MemcachedStatus `json:"status,omitempty"`
     }
+    ``` 
+- Execute the command `operator-sdk generate openapi` then you should be able to check the CRD updated with the cluster scope such as the following example. 
```

Comment:
add a comma `,` after the command before the "then":
"Execute the command `operator-sdk generate openapi`, then"

rephrase: you should be able to check that the CRD was updated with the cluster scope as in the following example: